tests/incremental/issue-114416-expect-unused-inside-impl-block.rs

```diff
@@ -0,0 +1,32 @@
+// revisions: rpass1
```

Comment:
Seems like it would be nice if it was possible to add `revisions: rpass1` to a UI test so that we don't have to duplicate tests like this one across `tests/incremetal` and `tests/ui`.

First, is there already a good way to avoid this duplication?

If not, would being able to drive an incremental test from a UI test be undesirable for some reason?

If desirable, I'd be interesting in adding support for using the `revisions: rpass1` annotation in a UI test. Do you have any tips on how I might best explore that (in a separate PR)?

Reply:
See what `tests/ui/dep-graph` files do. They have an `// incremental` directive for this.